models/DataObject/Objectbrick/Data/Dao.php

```diff
@@ -66,6 +95,13 @@ public function save(DataObject\Concrete $object)
             $getter = 'get' . ucfirst($fd->getName());
 
             if (method_exists($fd, 'save')) {
+                if (!DataObject\AbstractObject::isDirtyDetectionDisabled() && method_exists($this->model, 'isFieldDirty')) {
+                    // ownerNameList contains the dirty stuff
+                    if (!in_array("'" . $key . "'", $fieldNameList)) {
```

Comment:
`$db->quote($key);`?

Reply:
good catch, should assembled the same way as above (line 76)